nixos/modules/system/boot/networkd.nix

```
@@ -41,10 +44,23 @@ let
           "OtherChannels"
           "CombinedChannels"
           "RxBufferSize"
+          "RxMiniBufferSize"
+          "RxJumboBufferSize"
           "TxBufferSize"
+          "RxFlowControl"
+          "TxFlowControl"
+          "AutoNegotiationFlowControl"
+          "GenericSegmentOffloadMaxBytes"
+          "GenericSegmentOffloadMaxSegments"
         ])
         (assertValueOneOf "MACAddressPolicy" ["persistent" "random" "none"])
         (assertMacAddress "MACAddress")
+        (assertInt "TransmitQueues")
+        (assertRange "TransmitQueues" 0 4096)
```

systemd states that this (and `ReceiveQueues`) take have a lower bound of `1`, not `0`. why the difference?

---

it's a typo, thanks for catching it!

```suggestion
        (assertRange "TransmitQueues" 1 4096)
```